features.py

```diff
@@ -0,0 +1,104 @@
+"""
+Script to compute audio features for the audio recordings in the Eisenberg Collection.
+The script goes through the song titles in the metadata file and checks if there are
+audio file paths that contain the title. If there is a match, the features are computed
+and added to a dictionary with the NYUAD Archive reference as a key. Audio file paths
+that have been matched are removed from the list of paths, so that the audio recordings
+that haven't been matched are provided to the user at the end.
+"""
+
+import librosa #feature computation
+from pathlib import Path #directory traversal
+import pandas as pd #read metadata DataFrame
```

Comment:
Note that pandas 1.2.x seems to have some problems interfacing with deepdish, but 1.1.x and earlier work. This will be resolved with the requirements file mentioned in #4.